zcash_client_backend/src/fees.rs

```diff
@@ -114,7 +115,7 @@ impl ChangeValue {
 /// The amount of change and fees required to make a transaction's inputs and
 /// outputs balance under a specific fee rule, as computed by a particular
 /// [`ChangeStrategy`] that is aware of that rule.
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
Same here, `Proposal` components should be serialized using the protobuf encoding.